code/go/0chain.net/blobber/worker.go

```diff
@@ -46,35 +43,3 @@ func refreshPriceOnChain(ctx context.Context) {
 
 	}
 }
-
-// startRefreshSettings sync settings from blockchain
-func startRefreshSettings(ctx context.Context) {
-	const REPEAT_DELAY = 60 * 3 // 3 minutes
-	var err error
-	var b *zcncore.Blobber
-	for {
-		select {
-		case <-ctx.Done():
-			return
-		case <-time.After(REPEAT_DELAY * time.Second):
-			b, err = config.ReloadFromChain(common.GetRootContext(), datastore.GetStore().GetDB())
```

Comment:
This `ReloadFromChain` will help us to load setting changes from 0chain. Otherwise if users changed the setting through zboxcli, and commit to the 0chain, the settings would not be synced to the blobbers.

Reply:
Then should we keep only this functionality and remove the updateBlobber transaction ?